notebooks/206-vision-paddlegan-anime/206-vision-paddlegan-anime.ipynb

```diff
@@ -0,0 +1,614 @@
+   "metadata": {},
+   "source": [
+    "The average brightness is computed by a standard formula, see https://www.w3.org/TR/AERT/#color-contrast. To adjust the brightness, the difference in brightness between the source and destination (anime) image is computed and the brightness of the destination image is  adjusted based on that. The image is then converted to an 8-bit image.\n",
+    "\n",
+    "We copy these functions to the next cell, and will use them for inference on the IR model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b77c029-1112-4ad6-903a-141242def5e1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Copyright (c) 2020 PaddlePaddle Authors. Licensed under the Apache License, Version 2.0\n",
+    "\n",
+    "\n",
+    "def calc_avg_brightness(img):\n",
+    "    R = img[..., 0].mean()\n",
+    "    G = img[..., 1].mean()\n",
+    "    B = img[..., 2].mean()\n",
+    "\n",
+    "    brightness = 0.299 * R + 0.587 * G + 0.114 * B\n",
+    "    return brightness, B, G, R\n",
+    "\n",
+    "\n",
+    "def adjust_brightness(dst, src):\n",
+    "    brightness1, B1, G1, R1 = AnimeGANPredictor.calc_avg_brightness(src)\n",
+    "    brightness2, B2, G2, R2 = AnimeGANPredictor.calc_avg_brightness(dst)\n",
+    "    brightness_difference = brightness1 / brightness2\n",
+    "    dstf = dst * brightness_difference\n",
+    "    dstf = np.clip(dstf, 0, 255)\n",
+    "    dstf = np.uint8(dstf)\n",
+    "    return dstf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74bce1ee-f070-42b9-b92f-95e37b82ea02",
+   "metadata": {},
+   "source": [
+    "### Do Inference on IR Model\n",
+    "\n",
+    "Load the IR model, and do inference, following the same steps as for the PaddleGAN model. See the [OpenVINO Inference Engine API notebook](../002-openvino-api/002-openvino-api.ipynb) for more information about inference on IR models.\n",
+    "\n",
+    "The IR model is generated with an input shape that is computed based on the input image. If you do inference on images with different input shapes, results may differ from the PaddleGAN results. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7f0f83cb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load and prepare the IR model.\n",
+    "ie = IECore()\n",
+    "net = ie.read_network(ir_path)\n",
+    "exec_net = ie.load_network(net, \"CPU\")\n",
+    "input_key = next(iter(net.input_info.keys()))\n",
+    "output_key = next(iter(net.outputs.keys()))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "adc56b11",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 1. Load the image and convert to RGB\n",
+    "image = cv2.cvtColor(cv2.imread(str(image_path), flags=cv2.IMREAD_COLOR), cv2.COLOR_BGR2RGB)\n",
+    "\n",
+    "# Step 2. Transform the image (only resize and transpose are still required)\n",
```

Comment:
**Skip if redundant**

I understand that the images are transformed during model conversion to the IR format, and before running the inference?